sdk/messaging/azservicebus/client.go

```diff
@@ -75,34 +80,21 @@ func NewClientFromConnectionString(connectionString string, options *ClientOptio
 // func NewClientWithNamedKeyCredential(fullyQualifiedNamespace string, credential azcore.TokenCredential, options *ClientOptions) (*Client, error) {
 // }
 
-type clientConfig struct {
+type clientCreds struct {
 	connectionString string
-	credential       azcore.TokenCredential
+
 	// the Service Bus namespace name (ex: myservicebus.servicebus.windows.net)
 	fullyQualifiedNamespace string
-	tlsConfig               *tls.Config
-}
-
-func applyClientOptions(client *Client, options *ClientOptions) error {
-	if options == nil {
-		return nil
-	}
-
-	client.config.tlsConfig = options.TLSConfig
-	return nil
+	credential              azcore.TokenCredential
 }
 
-func newClientImpl(config clientConfig, options *ClientOptions) (*Client, error) {
+func newClientImpl(config clientCreds, options *ClientOptions) (*Client, error) {
```

Comment:
Is there a reason the client param is named `config` when it's now a cred?

Reply:
Fixed (it just lagged behind my type rename)